lib/src/wrappers/DesktopWrapper.tsx

```diff
@@ -1,77 +1,54 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import Popover, { PopoverProps } from '@material-ui/core/Popover';
 import { WrapperProps } from './Wrapper';
 import { StaticWrapperProps } from './StaticWrapper';
-import { makeStyles } from '@material-ui/core/styles';
 import { InnerMobileWrapperProps } from './MobileWrapper';
 import { WrapperVariantContext } from './WrapperVariantContext';
-import { IS_TOUCH_DEVICE_MEDIA } from '../constants/dimensions';
 import { KeyboardDateInput } from '../_shared/KeyboardDateInput';
-import { InnerDesktopPopperWrapperProps } from './DesktopPopperWrapper';
+import { InnerDesktopTooltipWrapperProps } from './DesktopTooltipWrapper';
+import { PickerPopper, ExportedPickerPopperProps } from '../_shared/PickerPopper';
 
-export interface InnerDesktopWrapperProps {
-  /**
-   * Popover props passed to material-ui Popover.
-   */
-  PopoverProps?: Partial<PopoverProps>;
-}
+export interface InnerDesktopWrapperProps extends ExportedPickerPopperProps {}
 
 export interface DesktopWrapperProps
   extends InnerDesktopWrapperProps,
     WrapperProps,
-    Partial<InnerMobileWrapperProps & InnerDesktopPopperWrapperProps & StaticWrapperProps> {}
-
-const useStyles = makeStyles({
-  popover: {
-    '&:focus': {
-      outline: 'auto',
-      [IS_TOUCH_DEVICE_MEDIA]: {
-        outline: 0,
-      },
-    },
-  },
-});
+    Partial<InnerMobileWrapperProps & InnerDesktopTooltipWrapperProps & StaticWrapperProps> {}
 
 export const DesktopWrapper: React.FC<DesktopWrapperProps> = ({
   open,
   children,
-  PopoverProps,
   onDismiss,
   DateInputProps,
   KeyboardDateInputComponent = KeyboardDateInput,
+  TransitionComponent,
+  PopperProps,
 }) => {
-  const ref = React.useRef<HTMLDivElement>(null);
-  const classes = useStyles();
+  const isFocusedRef = React.useRef(false);
+  const dialogRef = React.useRef<HTMLDivElement>(null);
+  const inputRef = React.useRef<HTMLDivElement>(null);
 
   return (
     <WrapperVariantContext.Provider value="desktop">
-      <KeyboardDateInputComponent {...DateInputProps} containerRef={ref} />
+      <KeyboardDateInputComponent {...DateInputProps} containerRef={inputRef} />
```

Comment:
Shouldn't listen to the "Input" rather than the "container"? The popup open bellow the helper text, I believe it should open below the `<input />` element.

Reply:
Maybe it's a good opportunity to solve #1768? I don't know.